OpenOversight/app/templates/list_unit.html

```diff
@@ -0,0 +1,190 @@
+{% extends "base.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block content %}
+<div class="container" role="main">
+  <h1>{{ department.name | title }} | {{ unit.descrip | title }} Unit | Officers</h1>
```

Comment:
want to change this to the other breadcrumbs, I think.

department Bar > unit Foo > officers